src/hooks/useMutation.ts

```diff
@@ -26,7 +26,9 @@ export const useMutation = <T = any, V = object>(
     data: undefined,
   });
 
-  const executeMutation = (variables?: V) => {
+  const executeMutation = (variables?: V, skip?: boolean) => {
+    if (skip) return;
```

Comment:
Same as the component; it's imperative so the addition doesn't really make sense 😀